### DISCUSSION TITLE
X88 Pro Pro X3 / X99 Max Plus / Transpeed X3 Plus.

### DISCUSSION THREAD

**Discussion by araczkowski** [ophub/amlogic-s9xxx-armbian#1589] [Amlogic]
I have X88 Pro X3 with S905x3 is it possible to install Armbian on it?
Does anybody did this?

More info about this box:
https://4pda.to/forum/index.php?showtopic=982302

**Comment by araczkowski** [comment]
I was able to run the Armbian on this box - I will write about this in separate issue

**Comment by petesimon** [comment]
@araczkowski Yes. In October, I installed this file _Armbian_24.11.0_amlogic_s905x3-x88-pro-x3_jammy_6.1.112_server_2024.10.02.img.gz_ from the 'releases' page. and it works great. the gigabit ethernet and wifi work. there is no bluetooth or sound, and only 2 GB RAM even if it should be 4 GB. i use the command `ethtool eth0` to check ethernet speed, and `nmcli d wifi list` to see list of available wifi connections. i connected a usb hard drive to the 3.0 port and raw write/read speeds are about 250 MB/s.
☺

![x88tvbox-owc-eliteraid1-dual-usb3-WD12TB-armbian-Ophub](https://github.com/user-attachments/assets/94dd2dc6-2ea1-4cec-8436-29afc979e96c)
